examples/window-app/nrfconnect/main/WindowCovering.cpp

```diff
@@ -35,10 +35,21 @@ using namespace chip::app::Clusters::WindowCovering;
 
 static const struct pwm_dt_spec sLiftPwmDevice = PWM_DT_SPEC_GET(DT_ALIAS(pwm_led1));
 static const struct pwm_dt_spec sTiltPwmDevice = PWM_DT_SPEC_GET(DT_ALIAS(pwm_led2));
-static k_timer sLiftTimer;
-static k_timer sTiltTimer;
+
 static constexpr uint32_t sMoveTimeoutMs{ 200 };
 
+static constexpr uint32_t FromOneRangeToAnother(uint32_t aInMin, uint32_t aInMax, uint32_t aOutMin, uint32_t aOutMax,
```

Comment:
This whole function need to be removed-> NRF app is almost forking everything rather than consolidating and refactorize with the EFR32 app.
The goal is to converge to an abstracted demo with the as much as common piece.
This made everything very hard to maintain.

Reply:
Ok, I see the problem, we made this change in the example inside our SDK, and I wanted to move it to the example inside connectedhomeip to be coherent within our examples. But now I can see that `Percent100thsToValue` function does the same actions as `FromOneRangeToAnother`, so, ok, I agree with you, it should be deleted.

Reply:
Even with your SDK , you will still have the Matter SDK to fetch along , right ?
Does your SDK can work without Matter SDK available ?
If not , you will always have this as decency anyway same goes to attribute accessor which are autogenerated

Reply:
Yes, We have a fork of Matter SDK inside our SDK, updated regularly. We did not notice that `Percent100thsToValue` is a core function, so we implemented a new one in SDK. We can use only Matter core functions because we don't want to use `common` functions from other examples. We must maintain our examples everywhere, and we try to keep them coherent.

Reply:
Percent100thsToValue  is defined inside app/clusters/window-covering-server, and should be as `core`

Reply:
Also you can propose things to put in the core if needed if exemple common is not the right place